package.json

```diff
@@ -20,9 +20,9 @@
     ]
   },
   "devDependencies": {
-    "@hapi/code": "8.x.x",
+    "@hapi/code": "9.x.x",
```

Comment:
Since we're updating the version, should we move it to npm's default range management aka caret?